library_generation/model/config_change.py

```diff
@@ -143,11 +146,22 @@ def __create_qualified_commit(
         :return: qualified commits.
         """
         libraries = set()
+        file_change_num = len(commit.stats.files.keys())
         for file in commit.stats.files.keys():
-            if file.endswith("BUILD.bazel"):
-                continue
             versioned_proto_path = find_versioned_proto_path(file)
             if versioned_proto_path in proto_paths:
+                # determine a commit that only contains `BUILD.bazel`
```

Comment:
I think the `if` statement this comment refers to _will skip the commit if the only affected file in the commit is the BUILD file and doesn't contain relevant changes_. Can you please explain that in the comment?

Reply:
Done.